src/util/stateGetter.js

```diff
@@ -10,15 +10,22 @@
 
 export const stateGetter = (state, props, key, entry) => {
 
-    if (props
-        && props.reducerKeys
-        && Object.keys(props.reducerKeys).length > 0
-        && props.reducerKeys[key]) {
+    if (props && props.reducerKeys) {
+        if (typeof props.reducerKeys === 'string' &&
+            props.reducerKeys.length > 0) {
```

Comment:
if `reducerKeys` is of type `string` and it's defined, won't it have a `length` > 0? Seems like even if you pass the reducerKey of `grid`, this check would return false?